### ISSUE TITLE
Encoding issue

### ISSUE THREAD

**Issue by Mathisca** [SilentSys/SCEP#3]
Hello, when i go on a steam page, this text : 
Étiquettes: Fusil, AUG, Collection Lake, Normal, Qualité artisanale, Testée sur le terrain, Échangeable, Négociable)
transform to 
Ã‰tiquettes: Fusil, AUG, Collection Lake, Normal, Qualité artisanale, Testée sur le terrain, Ã‰changeable, NÃ©gociable

**Comment by SilentSys** [comment]
Yeah, definitely an encoding issue. I will try to get it fixed for the next update.

Thanks!
